R/sprinkle.R

```diff
@@ -1,4 +1,4 @@
-#' @name sprinkles
+#' @name sprinkle
```

Comment:
You'll probably think it's silly, but I tend to keep the tags just so I can scan over them and see what I've been using in the function.  It helps me remember what did when I come back to something several months later.  You aren't aware of any issues in doing so are you (other than it being weird and kind of obsessive)?